@stellar/anchor-tests/src/tests/sep12/getCustomer.ts

```diff
@@ -187,12 +209,32 @@ export const canFetchExistingCustomerById: Test = {
     },
     ...genericFailures,
   },
-  async run(_config: Config): Promise<Result> {
+  async run(config: Config): Promise<Result> {
+    if (
+      !config.sepConfig ||
+      !config.sepConfig["12"] ||
+      !config.sepConfig["12"].customers ||
+      !config.sepConfig["12"].createCustomer ||
+      !config.sepConfig["12"].customers[config.sepConfig["12"].createCustomer]
+    ) {
+      throw new Error(
+        "SEP-12 configuration data is missing, expected a key within the " +
+          "'customers' object matching the value assigned to 'createCustomer'",
+      );
+    }
     const result: Result = { networkCalls: [] };
+    const customerType =
+      config.sepConfig["12"].customers[config.sepConfig["12"].createCustomer]
+        .type;
```

Comment:
NIT: similar thought and getting the customerType from the config. Since you have to do this lengthy drilling down into the object in all 3 cases, might be a little cleaner to have a helper method that returns `type`.